components/UI/navbar/index.js

```diff
@@ -16,6 +16,11 @@ const Navbar = ({ drawerToggleClicked }) => {
           <span />
         </div>
         <ul>
+          <li className="nav-logo-li">
```

Comment:
can we please use `module-css` here? like how it's done elsewhere in the whole project?